### PR TITLE
Update network throttling to simulate more realistic network conditions

### PR DESCRIPTION
**Pull request by wwwillchen**
Based on the work in https://github.com/GoogleChrome/lighthouse/pull/2183 which I'll land as a follow-up.

### CODE REVIEW

File: lighthouse-core/lib/emulation.js [hunk 1]
```diff
@@ -41,10 +41,24 @@ const NEXUS5X_USERAGENT = {
...
+/**
+ * Adjustments needed for DevTools network throttling to simulate
+ * more realistic network throttling conditions
+ */
+const LATENCY_FACTOR = 3.75;
+const THROUGHPUT_FACTOR = 0.9;
+
+const TARGET_LATENCY = 150; // 150ms
+const TARGET_DOWNLOAD_THROUGHPUT = Math.floor(1.6 * 1024 * 1024 / 8); // 1.6Mbps
+const TARGET_UPLOAD_THROUGHPUT = Math.floor(750 * 1024 / 8 * THROUGHPUT_FACTOR); // 750Kbps
```
Comment (patrickhulce) [inline lighthouse-core/lib/emulation.js]: why is the throughput factor included in the target here?

File: lighthouse-core/lib/emulation.js [hunk 2]
```diff
@@ -41,10 +41,24 @@ const NEXUS5X_USERAGENT = {
     '(KHTML, like Gecko) Chrome/59.0.3033.0 Mobile Safari/537.36'
 };
 
+/**
+ * Adjustments needed for DevTools network throttling to simulate
+ * more realistic network throttling conditions
```
Comment (patrickhulce) [inline lighthouse-core/lib/emulation.js]: should we link off to any explanations or references?

File: lighthouse-core/lib/emulation.js (hunk 1 above)
Comment (wwwillchen) [inline lighthouse-core/lib/emulation.js]: whoops - fixed.

File: lighthouse-core/lib/emulation.js (hunk 2 above)
Comment (wwwillchen) [inline lighthouse-core/lib/emulation.js]: Added a link to the crbug (not sure whether to make the analysis doc public yet)

**Pull request merged**